recipes/roman/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "roman" %}
+{% set version = "3.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/roman-{{ version }}.tar.gz
+  sha256: 2c46ac8db827d34e4fa9ccc0577e7f0b0d84f16ffe112351bd4f1ec2eb12d73f
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - roman=roman:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - python >=3.5
+
+test:
+  imports:
+    - roman
+  commands:
+    - pip check
+    - roman --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/zopefoundation/roman
+  license: ZPL-2.1
+  license_file: LICENSE.txt
```

Comment:
Would be good to have them package the license file and then simply pass that along here instead of including their license file in our recipe. Raised issue ( https://github.com/zopefoundation/roman/issues/9 )

Reply:
@thewchan can you please follow up with upstream and make sure this is being handled correctly? 🙂

Reply:
@jakirkham it looks like they changed their license from ZPL-2.1 to PFL-2.1 (Python software foundation license 2.1); I saw your PR was closed over there too. I once their repo goes through the deployment process the license will be included in the wheel file. Is there anything I need to change at the feedstock?

Reply:
Would try running `python setup.py sdist` and seeing if the license file gets packaged in the `sdist`. Also would raise an issue on the feedstock to switch to `PFL-2.1` on the next release